src/lib/common/string.cpp

```diff
@@ -676,3 +676,41 @@ void strReplace(char* to, int toLen, const char* from, const char* oldString, co
 
   to[toIx] = 0;
 }
+
+
+
+/* ****************************************************************************
+*
+* servicePathCheck - 
+*/
+std::string servicePathCheck(const char* servicePath)
```

Comment:
A bit surprised of seeing this as new code ("green" in githbu.com)... I though that we were already doing this servicePath checking. Maybe is was moved from another part I haven't seen so far?